content/22.vaccines.md

```diff
@@ -160,58 +160,114 @@ Modern technologies such as mRNA vaccines and vectored vaccines seem to have bee
+In the phase II trial, 482 participants were enrolled (3:1, vaccine:placebo).
+Participants in the vaccine condition received either a single 8 &mu;g dose or a double immunization of a 4 &mu;g/dose that was administered 14, 21, or 28 days post the prime dose.
+Participants in the placebo condition received the placebo on one of the same four schedules.
+The vaccine appeared well-tolerated, with 23% reporting at least one adverse reaction characterized as mild to moderate. 
+It was reported that all participants had a humoral immune response to the vaccines by day 42 but that the double immunization dosing regimen of 4 &mu;g/dose achieved higher neutralizing antibody titers than a single dose of 8 &mu;g and that the highest response was seen in the double-immunization regimen when at least 21 days separated the two doses [@doi:10/ghjkrf]. 
+Similar findings were reported in another phase I/II trial published by the same authors [@doi:10.1001/jama.2021.8565].
+
+In July 2020, phase III trials begin in the United Arab Emirates (UAE) and in Morocco and Peru a month later [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html]. 
+Sinopharm affiliates in the UAE in early December 2020 claimed the vaccine had 86% efficacy, which was later at odds with a Sinopharm Beijing affiliate that stated that the BBIBP-CorV vaccine had a 79.34% efficacy later that same month [https://www.reuters.com/article/health-coronavirus-china-vaccine-int-idUSKBN2940CA].
+China approved the vaccine for use shortly after and in May 2021 the WHO added the vaccine to their emergency use listing, citing a vaccine efficacy of 78.1% [@url:https://www.who.int/news/item/07-05-2021-who-lists-additional-covid-19-vaccine-for-emergency-use-and-issues-interim-policy-recommendations].
+Notably, a preprint reported that antisera from 12 people immunized with BBINP-CorV exhibited neutralizing antibody capacity against the beta variant (B.1.351), wild type SARS-CoV-2 (NB02), and one of the original variants of SARS-CoV-2 (D614G) [@doi:10.1101/2021.02.01.429069].
+Another preprint including sera from 282 participants used a surrogate neutralizing assay, a test that generally correlates with neutralizing antibodies, to determine that BBINP-CorV appears to induce neutralizing antibodies against the binding of the RBD of wild type SARS-CoV-2 and the alpha, beta, and delta variants to ACE2 [@doi:10.1101/2021.07.15.21260621].
+Indeed, a study in _The New England Journal of Medicine_ showed that the alpha variant exhibited very little resistance to neutralization by sera of those immunized with BBINP-CorV, but the beta variant was more resistant to neutralization by almost a factor of 3 [@doi:10.1056/NEJMc2103022]. 
+However, in May 2021, the UAE announced it would consider booster shots for all citizens who had been immunized with BBIP-CorV, which was shortly followed by a similar announcement in Bahrain. 
```

Comment:
```suggestion
However, in May 2021, the UAE announced it would consider booster shots for all citizens who had been immunized with BBIP-CorV, which was shortly followed by a similar announcement in Bahrain. 
```
Is BBIP correct or its it supposed to be BBIBP? (it's also used on the next few lines!)

Reply:
Corrected to BBIBP